doc/CodingStandard.md

```diff
@@ -884,6 +918,39 @@ if (0 == a && 0 != b) { ...
 end = (uint8_t*)start + size;
 ```
 
+#### Continuing expessions containing binary operators
+* When folding overly long lines, begin the next line with the binary operator.
+* Ensure the operators are aligned.
```

Comment:
I don't think suggesting alignment is necessarily a good idea. I think the second and subsequent lines should be indented as continued lines would be (at least one tab). We might include an example that shows how all parts might be aligned by starting with the identity of the binary operator:
```c
uint32_t result = 0
        + longVariableNameOrCalculation1
        + longVariableNameOrCalculation2
        + longVariableNameOrCalculation3;
```

Reply:
I'll remove the suggestion of alignment in favour of suggesting consistent indentation. Though, I think the example of:
```c
uint32_t result = longVariableNameOrCalculation1
            + longVariableNameOrCalculation2
            + longVariableNameOrCalculation3;
```
is sufficient.

Reply:
That's fine; my point, though, was that sometimes putting (what would otherwise be) the first operand on the next line is that it might clarify the relationship between the (originally) first operand and later operands.

Reply:
Ok, I understand. Despite similarity in naming, I wasn't trying to embed any type of (special) relationship between the `longVariableNameOrCalculation` operands. I think I'll leave it as described in my above comment.